### ISSUE TITLE
Custom CSS?

### ISSUE THREAD

**Issue by Jaxel** [guysoft/FullPageOS#246]
Is there a way to apply a custom CSS file?

**Comment by Jaxel** [comment]
Currently, I'm using a Chrome Extension called User CSS to apply custom CSS to my page... but Chrome Extensions don't work with FullPageOS.

Is there a solution for this?

**Comment by guysoft** [comment]
Yes they do, you can install chrome extensions. Get a UI by pressing ctrl+t
